crates/debugger_ui/src/debugger_panel.rs

```diff
@@ -580,9 +580,9 @@ impl DebugPanel {
                             pane.add_item(Box::new(tab), true, true, None, cx);
                         });
 
-                        if let Some(mut message_queue) = this.message_queue.remove(&client_id) {
-                            while let Some(output) = message_queue.pop_front() {
-                                cx.emit(DebugPanelEvent::Output((client_id, output)));
+                        if let Some(message_queue) = this.message_queue.get(&client_id) {
+                            while let Some(output) = message_queue.iter().next() {
+                                cx.emit(DebugPanelEvent::Output((client_id, output.clone())));
```

Comment:
We need a way to filter messages by thread_id to stop log messages from one thread outputting to another thread.

I also think it could be worth looking into sharing the same `View<Console>` between thread's of the same client_id

Reply:
Yeah so that is not really possible because the output does not belong to a thread it belongs to a client. We could add some indicator so you can see the logs since the thread stopped.

Yeah wanted to do that as well.